crates/resolver/src/name_server/name_server.rs

```diff
@@ -138,10 +138,15 @@ impl<C: DnsHandle<Error = ResolveError>, P: ConnectionProvider<Conn = C>> NameSe
         request: R,
     ) -> Result<DnsResponse, ResolveError> {
         let mut client = self.connected_mut_client().await?;
+        let now = std::time::Instant::now();
```

Comment:
Would prefer this to be just `Instant::now()`, unless there's some other type named `Instant` in scope here?